app/native/src/components/relay/QueryRenderer.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import * as React from 'react';
+import { QueryRenderer as OriginalQueryRenderer } from 'react-relay';
+import { FullPageLoading } from '@kiwicom/native-common';
+
+import createEnvironment from '../../services/relay/Environment';
+import GeneralError from '../../components/errors/GeneralError';
+
+import type { Props as PublicProps } from './PublicApiRenderer';
+
+type Props = {|
+  ...PublicProps,
+  accessToken?: string,
+|};
+
+export default class QueryRenderer extends React.Component<Props> {
+  renderRelayContainer = ({
+    error,
+    props,
+  }: {
+    error: Object,
+    props: Object,
+  }) => {
+    // FIXME: it Relay swallows all GraphQL errors?
```

Comment:
_Does_ Relay swallow 👮

Reply:
🤦‍♂️ This is what happens when you are rewriting just part of the message... 😄